src/Type/Integer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Integer value object class.
+ *
+ * Implemented as an immutable object with a pair of named constructors.
+ */
+
+namespace Joomla\Webservices\Type;
+
+final class Integer extends Type
```

Comment:
Yep, but which one?  I was tending towards "integer" on the basis of my pref. for avoiding unnecessary abbrevs. :stuck_out_tongue: